ts/src/header-validator/aggregatable-contributions.ts

```diff
@@ -24,7 +24,10 @@ function createAggregatableContributionsFromKeysAndValues(
     if (value === undefined) {
       continue
     }
-    contributions.push({ key, value })
+    contributions.push({
+      key,
+      value: typeof value === 'number' ? value : value.value,
```

Comment:
IIUC the `true` condition can never be reached as `value` is always an object now.